pkgs/shells/fish/plugins/fish-bd.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildFishPlugin,
+  fetchFromGitHub,
+  gitUpdater,
+}:
+buildFishPlugin rec {
+  pname = "fish-bd";
+  version = "1.3.3";
+
+  src = fetchFromGitHub {
+    owner = "0rax";
+    repo = "fish-bd";
+    rev = "v${version}";
+    hash = "sha256-GeWjoakXa0t2TsMC/wpLEmsSVGhHFhBVK3v9eyQdzv0=";
+  };
+
+  passthru.updateScript = gitUpdater { rev-prefix = "v"; };
+
+  meta = {
+    description = "Quickly go back to a parent directory up in your current working directory tree";
```

Description can be more consistent with [existing fish plugins](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=fishPlugins).

```suggestion
    description = "Fish plugin to quickly go back to a parent directory up in your current working directory tree";
```